lib/tasks/sync.rake

```diff
@@ -0,0 +1,23 @@
+namespace :foreman_inventory_upload do
```

Comment:
```suggestion
namespace :foreman_rh_cloud do
```

Reply:
Changed.
I used `foreman_inventory_upload` to keep all tasks close together on `foreman-rake -T`, so they are easier to discover. After this change, they are still close enough, and if user greps `foreman-rake -T` for "cloud", they will find all of them.

Thanks!